mathml/relations/css-styling/dynamic-dir-1.html

```diff
@@ -0,0 +1,100 @@
+<!DOCTYPE html>
+<html class="reftest-wait">
+<head>
+  <meta charset="utf-8">
+  <title>Test dynamically changing dir attribute</title></head>
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#cssproperties">
+  <link rel="help" href="https://mathml-refresh.github.io/mathml-core/#dom">
+  <meta name="assert" content="The dir attribute should update direction map to css properties dynamically">
+  <link rel="match" href="dynamic-dir-1-ref.html">
+  <script>
+      window.addEventListener("DOMContentLoaded", () => {
+        ["math", "mstyle", "mrow"].forEach((tag) => {
+          let elements = document.getElementsByTagName(tag)
+          
```

Comment:
wpt lint seems to be complaining that you have trailing whitespace on this line: https://tools.taskcluster.net/groups/DleV1p1mSiK4TPww_jTzXg/tasks/VkMWFadSSCKKG4Day3a0sA/runs/0/logs/public%2Flogs%2Flive.log#L37